log/error.log

```diff
@@ -0,0 +1,5 @@
+[12/May/2022:23:57:06 +0530] During processing of request from 127.0.0.1:57098
```

Comment:
Shouldn't log files be .gitignored?

Reply:
I saw errors that Snap was looking for these files. i have cleaned these, committed them and gitignored them.